packages/gestalt/src/IconButtonFloating.js

```diff
@@ -21,6 +21,10 @@ type Props = {|
    * String that clients such as VoiceOver will read to describe the icon button. Always localize the label. See [Accessibility section](https://gestalt.pinterest.systems/web/iconbuttonfloating#Accessibility) for more info.
    */
   accessibilityLabel: string,
+  /**
+   * When disabled, IconButtonFloating looks inactive and cannot be interacted with
```

Comment:
This description on copied from `IconButton` disabled prop, as you can see [Here](https://gestalt.pinterest.systems/web/iconbutton#IconButtonProps).